### PR TITLE
Working on creating start and stop timer for the main page

### PR DESCRIPTION
**Pull request by fancast**
I added the start and stop timer where the timer shows the hours, minutes, and seconds of time passed since the user clicked start.

### CODE REVIEW

File: fudge-ratio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
       <h1> Hello world!! Start files finished :-) </h1>
-      <p>Awesome</p>
+      <p>Awesome!</p>
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: This first 2 commits are gold 😂

File: fudge-ratio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,11 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Fudge Ratio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: Are we going to end up using bootstrap (I mean I like it) but I see no code that uses this, so why not add it when we will use it?

File: fudge-ratio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,11 +2,19 @@
...
+    <title>Fudge Ratio</title>
   </head>
-  <body>
-      <h1> Hello world!! Start files finished :-) </h1>
-      <p>Awesome!</p>
+  <body class="container">
+    <h1> Fudge Ratio Calculator </h1>
+    <p>Click the start button when you're ready!</p>
+    <br/><br/>
+    <button onclick="startTimer()">Start</button>
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: This is great, and is a perfectly valid option, but why clutter the HTML (given that HTML should be semantic and not mixed with styles and logic)?
Another option is to give the 2 buttons ID and the add the events listeners in Javascript

File: fudge-ratio/target/fudge_ratio-1/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SoyOscarRH) [inline fudge-ratio/target/fudge_ratio-1/WEB-INF/appengine-web.xml:1]: THIS IS NOT YOUR PROBLEM:
But when Giovanna created the repo she created a .git-ignore file, so we wont end up with files in the target folder, but I believe something went wrong, I do not thing there is much we can do beside delete them and check what is going on with the gitignore in another commit

File: fudge-ratio/src/main/webapp/index.html [hunk 5]
```diff
@@ -2,11 +2,19 @@
...
+    
+    <br/><br/>
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: I also like to use $< br / >$, I wonder if we could do it with just CSS (maybe if i have some free time I could do it)

File: fudge-ratio/src/main/webapp/index.html [hunk 6]
```diff
@@ -11,10 +11,11 @@
     <h1> Fudge Ratio Calculator </h1>
     <p>Click the start button when you're ready!</p>
     <br/><br/>
-    <button onclick="startTimer()">Start</button>
-    
-    <br/><br/>
-    <button onclick="stopTimer()">Stop</button>
+    <div id="timer">00:00:00</div>
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: great use of the id

File: fudge-ratio/src/main/webapp/index.html [hunk 7]
```diff
@@ -11,10 +11,11 @@
...
+    <div class="timer-buttons">
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/index.html]: great idea the div wrapping!

File: fudge-ratio/src/main/webapp/script.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
+
+
+var seconds = 0;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/script.js]: why are we using var? (here is a article on the pitfalls of var: https://levelup.gitconnected.com/stop-using-var-to-declare-variables-in-javascript-6c0caec16f43)

File: fudge-ratio/src/main/webapp/script.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    if(!timer) {
+        timer = setInterval(interval, 1000);
+    }
+}
+
+function stopTimer() {
+    
+}
+
+function addZero(number) {
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/script.js:38]: super useful helper function, love it😊

File: fudge-ratio/src/main/webapp/script.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    if(minutes == 60) {
+        hours++;
+        minutes = 0;
+    }
+}
+
+function startTimer() {
+    if(!timer) {
+        timer = setInterval(interval, 1000);
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/script.js:28]: great idea to store it in a variable so we dont leak memory

File: fudge-ratio/src/main/webapp/script.js [hunk 11]
```diff
@@ -17,15 +17,16 @@ function interval() {
...
 function startTimer() {
     if(!timer) {
         timer = setInterval(interval, 1000);
     }
 }
 
 function stopTimer() {
-    
+    clearInterval(timer);
+    timer = 0;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/script.js:34]: Clever, very clever

File: fudge-ratio/src/main/webapp/style.css [hunk 12]
```diff
@@ -1 +1,14 @@
-/* This is a css page for the index.html */
\ No newline at end of file
+/* This is a css page for the index.html */
+
+.container {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/style.css:8]: flexbox lover here! me too

File: fudge-ratio/src/main/webapp/style.css [hunk 13]
```diff
@@ -1 +1,14 @@
...
+  height: 100vh;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/style.css:9]: also, thanks for using vh, very interesting

File: fudge-ratio/src/main/webapp/style.css [hunk 14]
```diff
@@ -1 +1,14 @@
...
+}
+
+#timer {
+  font-size: 8em;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/style.css:13]: and for using em, you’ve surprising me

**Review by SoyOscarRH — COMMENTED** [review]
It’s great that you used many commits to split the work, I leave some comments, we can add a final commit and we will be done

**Review by SoyOscarRH — CHANGES_REQUESTED** [review]
The changes are the comments:
- event handles out of HTML
- use let and const instead of var (https://levelup.gitconnected.com/stop-using-var-to-declare-variables-in-javascript-6c0caec16f43)
- Remove bootstrap

File: fudge-ratio/src/main/webapp/index.html (hunk 1 above)
Comment (fancast) [reply fudge-ratio/src/main/webapp/index.html]: Haha, I was checking if my changes were local to my machine.

File: fudge-ratio/src/main/webapp/index.html (hunk 2 above)
Comment (fancast) [reply fudge-ratio/src/main/webapp/index.html]: That was a line of code I copied from my portfolio code. I'll remove it. I don't know what we will be using for front end. We can discuss suggestions in a meeting or in the group chat?

File: fudge-ratio/src/main/webapp/style.css (hunk 13 above)
Comment (fancast) [reply fudge-ratio/src/main/webapp/style.css:9]: No problem!

File: fudge-ratio/target/fudge_ratio-1/WEB-INF/appengine-web.xml (hunk 4 above)
Comment (fancast) [reply fudge-ratio/target/fudge_ratio-1/WEB-INF/appengine-web.xml:1]: I was not aware that a target folder was created. Good to know.

File: fudge-ratio/src/main/webapp/index.html (hunk 3 above)
Comment (fancast) [reply fudge-ratio/src/main/webapp/index.html]: I did it this way based on the tutorials I saw in the sps walkthroughs. I'll try using event listeners.

**Review by SoyOscarRH — APPROVED** [review]
Great!

**Comment by SoyOscarRH** [comment]
Time to merge! :D

**Pull request merged**